backend/utils/llm.py

```diff
@@ -178,7 +178,8 @@ def get_memory_summary(uid: str, memories: List[Memory]) -> str:
     Remember {user_name} is busy so this has to be very efficient and concise.
     Respond in at most 50 words.
   
-    Output your response in plain text, without markdown.
+    Output your response in plain text, without markdown. No newline character and only use numbers for the action items.
```

Comment:
I understand that we need. to fix the daily memory summary right? is 'get_memory_summary' func only used by the daily memory summary feature?

Reply:
Yes, I only found one reference for `get_memory_summary ` in the codebase